lib/twilio.rb

```diff
@@ -0,0 +1,12 @@
+require 'twilio-ruby'
+
+account_sid = "AC3dacdc49158a31763601f3cd87e88f48" # Your Account SID from www.twilio.com/console
```

Comment:
you can use environment variables to keep this sensitive information hidden

not entirely sure how to do this but you'd need another file to store them